### PR TITLE
Indiana National Guard and Reserve Pay Deduction 2023

### LINKED ISSUE #5031: Indiana National Guard and Reserve Pay Deduction 2023
IN military pay

[Reference
](https://www.in.gov/dor/tax-forms/2023-individual-income-tax-forms/)

### PR DESCRIPTION
**Pull request by srilareddy**
Fixes #5031

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 - bump: patch
   changes:
     fixed:
-    - Add switch to basic income phase-in.
+    - Indiana National Guard and Reserve Pay Deduction 2023
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Indiana National Guard and Reserve Pay Deduction 2023.
```

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+description: List of standard deductions applicable to Indiana taxpayers.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:1]:
```suggestion
description: Indiana provides the following deductions.
```

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+values:
+  2021-01-01:
+    - in_renters_deduction
+    - in_homeowners_property_tax_deduction
+    - salt_refund_last_year
+    - us_govt_interest
+    - tax_unit_taxable_social_security  # includes railroad retirement benefits
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:8]: move comment to new line in 2021 and 2023

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    - in_other_deductions
+    - military_basic_pay
+metadata: 
+  unit: list
+  Period: year
+  label: Indiana Deductions
+  reference:
+    -title: Indiana General Assembly Legal Code Taxation Title 6 
+     href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3
+
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:34]:
```suggestion
    - title: Indiana General Assembly Legal Code Taxation Title 6 
     href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3
 
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    - in_military_service_deduction
+    - in_nonpublic_school_deduction
+    - in_nol
+    - in_unemployment_compensation_deduction
+    - in_other_deductions
+
+  2023-01-01:
+    - in_renters_deduction
+    - in_homeowners_property_tax_deduction
+    - salt_refund_last_year
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:19]: could you cite subsections for each of these in comments?

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    - in_military_service_deduction
+    # IC 6-3-2-22-(d)(1)
+    - in_nonpublic_school_deduction
+    # IC 6-3-2-2.5
+    - in_nol
+    # IC 6-3-2-10
+    - in_unemployment_compensation_deduction
+    # IC 6-3-2
+    - in_other_deductions
+    # IT-40 Booklet-(621), pg. 22
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:36]: IC 6-3-2-4 (b)

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    - in_nol
+    - in_unemployment_compensation_deduction
+    - in_other_deductions
+
+  2023-01-01:
+    # IC 6-3-2-6
+    - in_renters_deduction
+    # IC 6-1.1
+    - in_homeowners_property_tax_deduction
+    # IT-40 Booklet 2023, pg.17
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:20]: IC 6-8-5-1

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    - salt_refund_last_year
+    # IT-40 Booklet 2023, pg.17
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:22]: IC 6-8-5-1

File: policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml (hunk 7 above)
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/in/tax/income/deductions/deductions.yaml:20]: IC 6-3.1-19-1

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**